tests/zstreamtest.c

```diff
@@ -2222,6 +2222,66 @@ static int basicUnitTests(U32 seed, double compressibility, int bigTests)
     }
     DISPLAYLEVEL(3, "OK \n");
 
+
+    DISPLAYLEVEL(3, "test%3i : Testing large offset with small window size: ", testNb++);
+    {
```

Comment:
We should only run this test when `--big-tests` is passed.

```suggestion
    if (bigTests) {
```

Reply:
It looks like `--big-tests` is false by default if the tests are built in 32-bit, but this test is specifically for 32-bit. Should we add a `--big-tests` flag to force enable big tests?

Reply:
Ah good catch - yeah lets add that flag, and add a CI job that runs zstreamtest with `--big-test` in CI.